pkgs/development/python-modules/unittest-data-provider/default.nix

```
@@ -0,0 +1,21 @@
+{ buildPythonPackage
+, lib
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  version = "1.0.1";
+  pname = "unittest-data-provider";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1gn2ka4vqpayx4cpbp8712agqjh3wdpk9smdxnp709ccc2v7zg46";
+  };
+
+  meta = with lib; {
+    description = "PHPUnit-like @dataprovider decorator for unittest";
+    homepage = https://github.com/yourlabs/unittest-data-provider;
+    license = licenses.free;
```

```suggestion
    license = licenses.mit;
```

https://github.com/yourlabs/unittest-data-provider/blob/891d9ea888fb9b1197c40391a0c5c8c91f23c3d3/setup.py#L19

---

hmm, guess I overlooked that, used to seeing a LICENSE file